stats.rst

```diff
@@ -0,0 +1,209 @@
+.. _stats:
+
+=================
+Stats Quick Start
+=================
+
+A quick guide to getting the internal stats collection working in flux.
+This guide assumes you have already built a flux binary; if you have not see :ref:`quickstart`.
```

Comment:
A larger summary here of what is meant by "stats" (e.g. counters, gauges, and timers) and a reference to StatsD might be useful to give a new user some context here.